### DISCUSSION TITLE
New Journey with Java

### DISCUSSION THREAD

**Discussion by Sakib99-cyber** [community/community#141628] [Programming Help]
### Body

Who started new work with **Java**?  Who works with **Java** from which country? 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by raghavg2004** [comment]
hii

**Reply by Sakib99-cyber** [reply]
Nice to meet you. which country ? @raghavg2004

**Reply by Sakib99-cyber** [reply]
I see, you are from India. I followed you, you can follow me. Although I have experience, I am new to GitHub. Come on, we can help ourselves in the future.  @raghavg2004
